src/components/EpisodeCard/EpisodeCard.spec.js

```diff
@@ -0,0 +1,10 @@
+import {render, screen} from '@testing-library/react'
+import EpisodeCard from './EpisodeCard'
+  
+describe('EpisodeCard', () => {
+    it('renders', () => {
+        render(<EpisodeCard />)
+        expect(screen.getByText('EpisodeCard')).toBeInTheDocument()
+    })
+})
+      
```

Comment:
maybe you could be more precise and test if the img is rendered for emample

Reply:
i have not yet implemented the test. That is just for having the file already.